services/webhook/slack.go

```diff
@@ -286,3 +286,10 @@ func GetSlackPayload(p api.Payloader, event webhook_model.HookEventType, meta st
 
 	return convertPayloader(s, p, event)
 }
+
+// IsValidSlackChannel validates a channel name conforms to what slack expects.
+// It makes sure a channel name cannot be empty and invalid ( only an # ).
+// The caller should call strings.TrimSpace first if needed.
+func IsValidSlackChannel(name string) bool {
+	return name != "" && name != "#"
```

Comment:
```suggestion
	return name != "" && name != "#" && strings.TrimSpace(name) == name
```

Reply:
Since you suggest being a little more strict regarding validation, I went all the way to strictly follow [Slack's guidelines](https://api.slack.com/methods/conversations.rename#naming)

> Conversation names can only contain lowercase letters, numbers, hyphens, and underscores, and must be 80 characters or less. We will validate the submitted channel name and modify it to meet the above criteria. When calling this method, we recommend storing the channel's name value that is returned in the response.